run_wav.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Run as script:
+
+$ python run_wav.py --codec=LSB_wav \
```

Comment:
```suggestion
$ python run_wav.py --codec=lsb_wav \
```

下面 `__init__.py` 里改了这里没改